app/src/ui/changes/no-changes.tsx

```diff
@@ -15,11 +15,11 @@ export class NoChanges extends React.Component<INoChangesProps, void> {
     return (
       <div className='panel blankslate' id='no-changes'>
         <img src={BlankSlateImage} className='blankslate-image' />
-        No local changes
+        <div>No local changes</div>
```

Comment:
We should only be using divs as a way to refer to elements or to group them together. I don't have strong opinions on this though. This is something I need clarification on as well, maybe would be useful to talk about in a couple of weeks.